README.md

```diff
@@ -44,6 +44,8 @@ The root object must have properties named `models` and `endpoints`, which are a
     - **name**: (Required) A unique string name for the endpoint following the pattern `^[a-z0-9-]+$`.
     - **integration**: (Required) Defines how the endpoint integrates with other services. (Same as the integration in models)
 
+:warning: `Caller is not subscribed to the marketplace offering` Before deploying, you may need to manually subscribe to models from the Marketplace [**here**](https://us-east-1.console.aws.amazon.com/sagemaker/home?region=us-east-1#/marketplace-search-model-packages!mpSearch/). :warning:
```

Comment:
- Remove the error message
- Should probably instead be worded "Depending on the model_package_arn you specify you may be required to subscribe to a Marketplace Offering to get access to the model package.
- Link to https://aws.amazon.com/marketplace instead of the console view, as console will re-route to marketplace when you select a model
- Remove the second warning

Reply:
- The error message will help users search the page for a resolution
- This is a better place to start to find the model you need to subscribe to since Sagemaker console filters down to just the maketplace models.

Reply:
You can set the same filters on marketplace like so:

 https://aws.amazon.com/marketplace/search/results?FULFILLMENT_OPTION_TYPE=SAGEMAKER_MODEL&filters=FULFILLMENT_OPTION_TYPE 

This doesn't require a user to be logged into the console to view models, and doesn't hardcode a region into their console through the url.

---

If you want to include an error message that should be added in a separate troubleshooting section at the bottom, this is just meant to be a note to users that they should subscribe to marketplace models in the first place which would avoid that error being raised.

Reply:
nice, that URL is better
ok, TODO: troubleshooting section